test/e2e/trafficcontrol_test.go

```diff
@@ -0,0 +1,254 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"antrea.io/antrea/pkg/apis/crd/v1alpha2"
+	agentconfig "antrea.io/antrea/pkg/config/agent"
+)
+
+var (
+	vni          = int32(1)
+	dstVXLANPort = int32(1111)
+	labels       = map[string]string{"tc-e2e": "agnhost"}
+
+	testNode string
+
+	testPodName          = "test-tc-pod"
+	testPodIP            = ""
+	testCollectorPodName = "test-packets-collector-pod"
+	testCollectorPodIP   = ""
```

Comment:
These global variables are too generic to conflict in this package if all tests declare their global variables in such way. It should either pass them as local variables to subtests or define a test struct for trafficControl like:
```
type trafficControlTestConfig {
  nodeName string
  podName string
  podIP string
  ...
}
```